log/docs/api/robocorp.log.md

```diff
@@ -219,7 +219,7 @@ ______________________________________________________________________
 console_message(
     message: str,
     kind: str,
-    stream: Union[IO, NoneType, _SentinelUseStdout] = <robocorp.log._SentinelUseStdout object at 0x10585afb0>,
+    stream: Union[IO, NoneType, _SentinelUseStdout] = <robocorp.log._SentinelUseStdout object at 0x104fab5e0>,
```

Comment:
Humm, that'll change on each run -- maybe you can change the `_SentinelUseStdout` in this PR so that its `__repr__` and `__str__` returns `"_SentinelUseStdout"` so that it's possible to do the docs check and get stable results here?

Reply:
sure can!